Python/ceval.c

```diff
@@ -4777,6 +4772,96 @@ positional_only_passed_as_keyword(PyThreadState *tstate, PyCodeObject *co,
 
 }
 
+static inline unsigned char *
```

Comment:
It would be worth adding a comment here that points back to exception_table_notes.txt.